.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: no-commit-to-branch
         args: [--branch, develop, --branch, main]
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.6.5"
+    rev: "v0.6.6"
```

Comment:
This is a regression, main has these changes since https://github.com/great-expectations/cloud/pull/449